portfolio/src/main/webapp/index2.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
Nit: there's a lot of new line breaks that seem extra. Just something to consider